server/channels/app/expirynotify.go

```diff
@@ -24,7 +24,7 @@ func (a *App) NotifySessionsExpired() error {
 	// Get all mobile sessions that expired within the last hour.
 	sessions, err := a.ch.srv.Store().Session().GetSessionsExpired(OneHourMillis, true, true)
 	if err != nil {
-		a.CountNotificationReason(model.NotificationStatusError, model.NotificationTypePush, model.NotificationReasonFetchError)
+		a.CountNotificationReason(model.NotificationStatusError, model.NotificationTypePush, model.NotificationReasonFetchError, "")
```

Comment:
Since this is for mobile, should this also track the client's type, or do we not care about it here?

Reply:
At this point we haven't yet extracted the platform (we haven't called setDeviceIdAndPlatform). Also, this part has nothing to do with the phone but on server code, so it shouldn't be necessary.

Reply:
I think the gist here is: we have an error for server not being able to get *all sessions*. To be fair, I'm not sure if it's right thing to track this failure if the metric is designated to have a **platform** (which indicates it's a sample should be per-client). Thoughts?

Reply:
The point here is that some errors have more information (the platform) and some don't have that information (we haven't reached the platform yet).

With the current approach is complicated to divide them into two different kind of metrics, because in any case the metric is... "number of times the notifications wasn't sent".